eng/pipelines/bompreparation.yml

```
@@ -0,0 +1,30 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - eng/pipelines/bompreparation.yml
```

```suggestion
pr: none
```
We don't want this ever being triggered other than manually.